presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestJdbcClient.java

```diff
@@ -111,4 +112,17 @@ public void testMetadataWithFloatAndDoubleCol()
                 new JdbcColumnHandle("COL3", JDBC_DOUBLE, DOUBLE),
                 new JdbcColumnHandle("COL4", JDBC_REAL, REAL)));
     }
+
+    @Test
+    public void testConvertRangeInfoIntoIntoPredicate()
+    {
+        RangeInfo range = new RangeInfo("id", Optional.empty(), Optional.of(10));
+        assertEquals(convertRangeInfoIntoIntoPredicate(range), "id < 10");
```

Comment:
convertRangeInfoIntoIntoPredicate, is this a typo with Into twice?

Reply:
typo, will fix that